hydrocarbon_problem/AspenSimulation/HydrocarbonMixture$backup.bkp

```diff
@@ -0,0 +1,724 @@
+MM "37.0" FLAVOR "NO" VERSION "37.0" DATETIME "Thu Mar 24 16:32:00 2022" 
```

Comment:
Are all of these files needed? If not remove them and only keep the bkp file that we need for the flowsheet.